File: Array/2942. Find Words Containing Character/findWordsContaining.go
func findWordsContaining(words []string, x byte) []int {
	result := []int{}
	for i, word := range words {
		for j := 0; j < len(word); j++ {
			if word[j] == x {
				result = append(result, i)
				break
			}
		}
	}
	return result
}

File: Array/2942. Find Words Containing Character/findWordsContaining.rs
impl Solution {
    pub fn find_words_containing(words: Vec<String>, x: char) -> Vec<i32> {
        let mut result = Vec::new();
        for (i, word) in words.iter().enumerate() {
            if word.contains(x) {
                result.push(i as i32);
            }
        }
        result
    }
}